### ISSUE TITLE
New test

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#22608]
I want to create a layout with 4 charts, but it doesn't work in Lynx.

Send help, and please provide an ETA for the fix.
